crates/yas_core/src/tests/test_contracts/test_yas_pool.cairo

```diff
@@ -486,81 +491,1014 @@ mod YASPoolTests {
+            };
+            use yas_core::contracts::yas_router::{
+                YASRouter, IYASRouterDispatcher, IYASRouterDispatcherTrait
+            };
+
+
+            #[test]
+            #[available_gas(2000000000)]
+            #[should_panic(expected: ('TLU', 'ENTRYPOINT_FAILED', 'ENTRYPOINT_FAILED'))]
+            fn test_fails_tick_lower_greater_than_tick_upper() {
+                let (yas_pool, token_0, token_1, mint_callback, min_tick, max_tick) = setup();
+                mint_callback
+                    .mint(
+                        yas_pool.contract_address,
+                        WALLET(),
+                        IntegerTrait::<i32>::new(1, false),
+                        IntegerTrait::<i32>::new(0, false),
+                        1
+                    );
+            }
+
+
+            #[test]
+            #[available_gas(2000000000)]
+            #[should_panic(expected: ('TLM', 'ENTRYPOINT_FAILED', 'ENTRYPOINT_FAILED'))]
+            fn test_fails_tick_lower_than_min() {
+                let (yas_pool, token_0, token_1, mint_callback, min_tick, max_tick) = setup();
+                mint_callback
+                    .mint(
+                        yas_pool.contract_address,
+                        WALLET(),
+                        IntegerTrait::<i32>::new(887273, true),
+                        IntegerTrait::<i32>::new(0, false),
+                        1
+                    );
+            }
+
+
+            #[test]
+            #[available_gas(2000000000)]
+            #[should_panic(expected: ('TUM', 'ENTRYPOINT_FAILED', 'ENTRYPOINT_FAILED'))]
+            fn test_fails_tick_greater_than_max() {
+                let (yas_pool, token_0, token_1, mint_callback, min_tick, max_tick) = setup();
+                mint_callback
+                    .mint(
+                        yas_pool.contract_address,
+                        WALLET(),
+                        IntegerTrait::<i32>::new(0, false),
+                        IntegerTrait::<i32>::new(887273, false),
+                        1
+                    );
+            }
+
+
+            #[test]
+            #[available_gas(2000000000)]
+            #[should_panic(expected: ('LO', 'ENTRYPOINT_FAILED', 'ENTRYPOINT_FAILED'))]
+            fn test_fails_amount_greater_than_max() {
+                let (yas_pool, token_0, token_1, mint_callback, min_tick, max_tick) = setup();
+                let grater_than_max_amount: u128 = yas_pool.max_liquidity_per_tick() + 1;
+                mint_callback
+                    .mint(
+                        yas_pool.contract_address,
+                        WALLET(),
+                        min_tick + IntegerTrait::<i32>::new(tick_spacing(FeeAmount::MEDIUM), false),
+                        max_tick - IntegerTrait::<i32>::new(tick_spacing(FeeAmount::MEDIUM), false),
+                        grater_than_max_amount
+                    );
+            }
```

Comment:
I suggest using the same names as in the Uniswap tests, as we have been doing. It helps make it more transparent and easy to compare.
```suggestion
                let max_liquidity_gross = yas_pool.max_liquidity_per_tick();
                mint_callback
                    .mint(
                        yas_pool.contract_address,
                        WALLET(),
                        min_tick + IntegerTrait::<i32>::new(tick_spacing(FeeAmount::MEDIUM), false),
                        max_tick - IntegerTrait::<i32>::new(tick_spacing(FeeAmount::MEDIUM), false),
                        max_liquidity_gross + 1
                    );
            }
```